nuxt.config.ts

```diff
@@ -1,4 +1,8 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
-  devtools: { enabled: true }
+  ssr: false,
+  modules: ['@pinia/nuxt', '@vueuse/nuxt', '@nuxt/ui', 'nuxt-icon'],
+  experimental: { typedPages: false },
```

Comment:
There is no mention of `typedPages` being enabled by default in the [docs](https://nuxt.com/docs/guide/going-further/experimental-features#typedpages).
Is there a reason for explicitly disabling this?

Reply:
Nice catch! :clap: 
This is actually [`false` by default](https://nuxt.com/docs/api/nuxt-config#typedpages) and I meant to enable it but messed up. Well done spotting and mentioning it, thanks!!